packages/amplify-e2e-tests/src/examples/ts-data-schema-validation/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "typebeast-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
minor nit: Amplify projects usually use MIT -- is this the right license for an embedded test project?